fairroot/base/sim/FairMCApplication.cxx

```diff
@@ -391,6 +391,7 @@ void FairMCApplication::FinishRun()
 
     if (!fRadGridMan && fRootManager) {
         fRootManager->Write();
+        fRootManager->CloseSink();
     }
```

Comment:
hm, so `!fRadGridMan && fRootManager` means MT mode?

how does it relate to `fMC->IsMT()`?